python/taichi/lang/matrix.py

```diff
@@ -1755,6 +1745,83 @@ def field(self, **kwargs):
         return Matrix.field(self.n, self.m, dtype=self.dtype, **kwargs)
 
 
+class VectorType(MatrixType):
+    def __init__(self, n, dtype):
+        super().__init__(n, 1, dtype)
+
+    def __call__(self, *args):
+        """Return a vector matching the shape and dtype.
+
+        This function will try to convert the input to a `n x m` matrix, with n, m being
```

Comment:
Should we replace `matrix` with `vector` in docstrings?

Reply:
nice catch, thx